CMake/SlicerMacroBuildApplication.cmake

```diff
@@ -307,11 +310,15 @@ macro(slicerMacroBuildApplication)
 
   set(expected_defined_vars
     NAME
-    LAUNCHER_SPLASHSCREEN_FILE
     APPLE_ICON_FILE
     WIN_ICON_FILE
     LICENSE_FILE
     )
+  if(NOT SLICERAPP_SPLASHSCREEN_DISABLED)
```

Comment:
I much prefer using positive terms for variable names (SLICERAPP_SPLASHCREEN_ENABLED instead of SLICERAPP_SPLASHSCREEN_DISABLED) to avoid dispersing error-prone and hard-to-read double negatives (NOT ..._DISABLED).

Would it be possible to use SLICERAPP_SPLASHCREEN_ENABLED variable name and initialize it to TRUE by default?

Reply:
Since the function `cmake_parse_arguments` accept option, and specifying option means the variable will be set to True, following this approach was sensible and consistent with how the launcher itself is configured.

https://github.com/commontk/AppLauncher/blob/1367de4c6efde0c11e87835fb7245ea2b05074aa/CMake/ctkAppLauncher.cmake#L436

Reply:
In  the Slicer macro, I could do something like this:

```
set(_spashscreen_enabled 1)
if(SLICERAPP_SPLASHSCREEN_DISABLED)
  set(_spashscreen_enabled 0)
endif()
```

and then test the variable `_spashscreen_enabled`. That seems to be a reasonable tradeoff ?

Reply:
The proper solution would be to be able to specify default value for undefined variables. I thought that this is already possible or easy to add, but if it is not then it is fine to keep the SLICERAPP_SPLASHSCREEN_DISABLED variable name as is (no need for introducing the local variable).

Reply:
I could indeed switch from having `SPLASHSCREEN_DISABLED` specified as an option (meaning if specified it is `ON` and `OFF` otherwise) to having a single arg parameter.

After switching to a single arg parameter, it will indeed be possible to have a default value. I will do that.